### ISSUE TITLE
403 in Job

### ISSUE THREAD

**Issue by lazee** [math-nao/certs#18]
Hi,

I'm having some issues that I hope you can help me solve.

```
I 2019-12-10T13:54:22.566587491Z wait few seconds in case ingress rule is deployed at the same as it is in demo
 
I 2019-12-10T13:54:52.569763971Z Info: Initialize environment...
 
I 2019-12-10T13:54:52.617841989Z Info: Invalid status code found: 403
 
````
I'm getting this error in the Kubernetes Job.

I'm using Domeneshop.no as my ISP. This is the yaml

```
spec:
  activeDeadlineSeconds: 600
  containers:
  - env:
    - name: ACME_MANAGE_ALL_NAMESPACES
      value: "true"
    - name: ACME_DEMO_ENABLED
      value: "false"
    - name: ACME_DEBUG
      value: "true"
    - name: DOMENESHOP_Token
      value: mytokenhere
    - name: DOMENESHOP_Secret
      value: mysecrethere
    image: mathnao/certs:1.1.3
    imagePullPolicy: Always
    name: certs
    resources: {}
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: certs-token-p4zj8
      readOnly: true
```

I'm fully able to generate the certificates manually with `acme.sh` from the repo https://github.com/Neilpang/acme.sh, and with the exact same tokens.

**Comment by lazee** [comment]
Any idea on how I can debug this issue?

**Comment by math-nao** [comment]
Hi, it seems to be a k8s api permission issue. I updated code to generate more logs. Could you update your values.yaml file to use `dev` docker image tag and try again?

`image:`
&nbsp;&nbsp;`tag: dev`

**Comment by p-v-a** [comment]
I ran into the same issue. You haven't published dev release, so I can't deploy debug version. However, I think I reproduced issue with just curl command ran from my pc.

HTTP/2 403
cache-control: no-cache, private
content-type: application/json
x-content-type-options: nosniff
content-length: 393
date: Thu, 02 Jan 2020 03:22:12 GMT

{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {

  },
  "status": "Failure",
  "message": "ingresses.extensions is forbidden: User \"system:serviceaccount:kube-system:cert-certs\" cannot list resource \"ingresses\" in API group \"extensions\" at the cluster scope",
  "reason": "Forbidden",
  "details": {
    "group": "extensions",
    "kind": "ingresses"
  },
  "code": 403
}

It works if you change Role to ClusterRole and RoleBinding to ClusterRoleBinding.
